homeassistant/components/growatt_server/const.py

```diff
@@ -8,13 +8,13 @@
 DEFAULT_NAME = "Growatt"
 
 SERVER_URLS = [
-    "https://server-api.growatt.com/",
+    "https://server.growatt.com/",
     "https://server-us.growatt.com/",
     "http://server.smten.com/",
 ]
 
 DEPRECATED_URLS = [
-    "https://server.growatt.com/",
+    "https://server-api.growatt.com/",
```

Comment:
This used to be the default, and was previously put into the config entries as well.

Should we migrate existing config entries to update the stored URL from `https://server.growatt.com/` to `https://server-api.growatt.com/`?

../Frenck

Reply:
This is not the correct change to make. The server.growatt.com url was deprecated because that's not what the android api uses anymore and is actively blocked by the underlying library.
The 'correct' workaround for this problem is the change that's outlined in this PR: https://github.com/home-assistant/core/pull/103542 where the alternative `openapi` url is being added instead.
Remember that the underlying library uses the Android API endpoints to work (which changed to `server-api` some while ago) not the URL that users access in their browsers.